CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [v0.22.2] - 2024-03-05
+
+This is a small patch release that exposes the connection details to server methods without breaking changes.
+Currently, users can only retrieve the connection ID from these details.
```

Comment:
I think it's better to omit this line because connection ID can be extracted by middleware etc...

```suggestion
```